### ISSUE TITLE
Some mods can't be found at all despite supporting current version

### ISSUE THREAD

**Issue by Xaphiosis** [PolyMC/PolyMC#1059]
### Operating System

Linux

### Version of PolyMC

PolyMC 1.4.1

### Version of Qt

Qt 5.12.8

### Description of bug

I'm running MC 1.19.2 with Fabric loader.
Doing a search for some Modrinth/Curseforge mods that definitely exist and work, I cannot find them even when setting the search filter to "Any Version".
Attempting to install manually and then re-acquire meta data via "Check for Updates" does not work either.
If the mod exists, it should be possible to find it with a "Any Version" filter! 

Preferably it would then be possible to force an "incompatible" install as well, but I believe there are other issues already covering that.

Note that this is different to finding a mod but being unable to select any version when (PolyMC concludes) there are no compatible versions.

### Steps to reproduce

View Mods -> Download Mods -> Modrinth -> Filter Options -> Any Version

Search for "Indium". No results. However https://modrinth.com/mod/indium exists and has working 1.19 version.
Search for "Clear Skies". Two non-relevant results. However https://modrinth.com/mod/clear-skies exists and has working 1.19 version.

### Suspected cause

It is possible that the "Any version" filter does not actually do anything. Looking at the console, the search looks like this:

```
 2838.612 D Task "NetJob(Modrinth::Search)" starting for the first time
 2838.612 D Downloading  "https://api.modrinth.com/v2/search?offset=0&limit=25&query=clear skies&index=relevance&facets=[[%22categories:fabric%22],[%22versions:1.19.2%22],[%22project_type:mod%22]]"
 2838.844 D Download succeeded: "https://api.modrinth.com/v2/search?offset=0&limit=25&query=clear skies&index=relevance&facets=[[%22categories:fabric%22],[%22versions:1.19.2%22],[%22project_type:mod%22]]
```

Which would indicate that it's requesting 1.19.2 from the API even though I selected "Any version"

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by IkeVoodoo** [comment]
I am having this as well, would be nice to have a way to tell the auto updater what version I want (strict, major, any)

**Comment by DioEgizio** [comment]
this was fixed in 1.4.2. this issue can be closed.

**Comment by flowln** [comment]
Fixed in https://github.com/PolyMC/PolyMC/pull/997 / https://github.com/PolyMC/PolyMC/pull/1118 :)
